cleanlab/segmentation/rank.py

```diff
@@ -0,0 +1,240 @@
+# Copyright (C) 2017-2023  Cleanlab Inc.
+# This file is part of cleanlab.
+#
+# cleanlab is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# cleanlab is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with cleanlab.  If not, see <https://www.gnu.org/licenses/>.
+
+"""
+Methods to rank and score images in a semantic segmentation dataset , based on how likely they are to contain label errors.
+
+
+"""
+
+import pandas as pd
+import numpy as np
+from typing import List, Optional, Union, Tuple
+
+from cleanlab.segmentation.filter import find_label_issues
+
+
+def get_label_quality_scores(
+    labels: np.ndarray,
+    pred_probs: np.ndarray,
+    method: str = "softmin",
+    batch_size: int = 10000,
+    n_jobs: Optional[int] = 1,
+    verbose: bool = True,
+    **kwargs
+) -> Tuple[np.ndarray,np.ndarray]:
+    """Returns a label quality score for each image.
+
+    This is a function to compute label quality scores for standard (multi-class) classification datasets,
+    where lower scores indicate labels less likely to be correct.
+
+    Score is between 0 and 1.
+
+    1 - clean label (given label is likely correct).
+    0 - dirty label (given label is likely incorrect).
+
+    Parameters
+    ----------
+    labels : np.ndarray 
+      A discrete array of noisy labels for a classification dataset, i.e. some labels may be erroneous.
+      *Format requirements*: for dataset with K classes, each pixel must be integer in 0, 1, ..., K-1.
+      For a standard (multi-class) classification dataset where each example is labeled with one class,
+      `labels` should be 3-D array of shape ``(N,H,W,)``. 
+      
+    Tip: If your labels are one hot encoded you can `np.argmax(labels_one_hot,axis=1)` assuming that `labels_one_hot` is of dimension (N,K,H,W)
+    before entering in the function
+
+    pred_probs : np.ndarray
```

Comment:
Keep explanation minimal and add:
Refer to documentation for this argument in :py:func:`find_label_issues <cleanlab.segemntation.filter.find_label_issues>` for further details.